Gopkg.toml

```diff
@@ -68,6 +68,10 @@ required = [
   branch = "master"
   name = "github.com/giantswarm/k8sportforward"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/giantswarm/e2etemplates"
+
```

Comment:
Nit but I'd like the ordering to be alphabetical.

Reply:
Should be alphabetical now.